src/components/pages/ForgotPassword.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { withRouter } from 'react-router-dom';
+import { ResetAction } from '../../actions/resetPasswordAction';
+import Input from '../global/Input';
+import Button from '../global/Button';
+
+export class Forgotpassword extends React.Component {
+    state = {
+      email: '',
+      isLoading: false,
+    };
+
+    // eslint-disable-next-line camelcase
+    UNSAFE_componentWillReceiveProps() {
```

Comment:
You do not this lifecycle method, you can set the isLoading from the `handleSubmit`